galeracluster/source/streamingreplication.rst

```diff
@@ -0,0 +1,90 @@
+
+Using :term:`Streaming Replication` on long-running transactions mitigates this situation.  Once the node replicates and certifies a fragment, it is no longer possible for other transactions to abort it.
+
+
+-----------------------------
+Large Data Write Transactions
+-----------------------------
+.. _`large-write-trx`:
+
+When using normal replication, the node locally processes the transaction and doesn't replicate the data until you commit.  This can create problems when updating a large volume of data, especially on nodes with slower network connections.
+
+Additionally, while slave nodes apply a large transaction, they cannot commit other transactions they receive, which may result in Flow Control throttling of the entire cluster.
+
+With :term:`Streaming Replication`, the node begins to replicate the data with each transaction fragment, rather than waiting for the commit.  This allows you to spread the replication over the lifetime of the transaction.
+
+In the case of the slave nodes, after the slave applies a fragment, it's free to apply and commit other, concurrent transactions without blocking.  This allows the slave node to process incrementally the entire large transaction with a minimal impact on the cluster.
+
+
+-----------
+Hot Records
+-----------
+.. _`hot-records`:
+
+In situations in which an application frequently updates one and the same records from the same table (e.g., when implementing a locking scheme, a counter, or a job queue), you can use :term:`Streaming Replication` to force critical updates to replicate to the entire cluster.
+
+Running a transaction in this way effectively locks the hot record on all nodes, preventing other transactions from modifying the row.  It also increases the chances that the transaction will commit successfully and that the client in turn will receive the desired outcome. 
+
+.. note:: For more information and an example of how to implement Streaming Replication in situations such as this, see :ref:`Using Streaming Replication with Hot Records <usr-hot-records>`.
+
+===========
+Limitations
+===========
+.. _`sr-limit`:
+
+In deciding whether you want to use :term:`Streaming Replication` with your application, consider the following limitations.
+
+----------------------------------
+Performance During a Transaction
+----------------------------------
+.. _`limit-in-trx`:
+
+When you enable :term:`Streaming Replication`, each node in the cluster begin recording its write-sets to the ``SR`` table in the ``wsrep_schema`` database.  The nodes do this to ensure the persistence of Streaming Replication updates in the event that they crash.  However, this operation increases the load on the node, which may adversely affect its performance.
```

Comment:
`SR` table in `wsrep_schema` was renamed to `wsrep_streaming_log` and resides in `mysql` database.